Dockerfile.rhel7

```diff
@@ -0,0 +1,20 @@
+FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.16-openshift-4.10 AS builder
+WORKDIR /go/src/github.com/openshift/secondary-scheduler-operator
+COPY . .
+
+RUN go build -o secondary-scheduler-operator ./cmd/cluster-kube-descheduler-operator
```

Comment:
Any particular reason we're not using `make build`, I've noticed we're doing similar in descheduler-operator?